src/DataGrid.tsx

```diff
@@ -0,0 +1,30 @@
+import { DataGrid as MaterialDataGrid, DataGridProps } from "@mui/x-data-grid";
+import { ReactElement } from "react";
+import { Card } from "./Card";
+
+interface Props extends DataGridProps {
+  title: string;
+  pageSize?: number;
+  rowsPerPageOptions?: number[];
+  checkboxSelection?: boolean;
+  disableSelectionOnClick?: boolean;
+}
+
+export function DataGrid(props: Props): ReactElement {
+  return (
+    <Card title={props.title} noPadding>
+      <MaterialDataGrid
+        sx={{ height: "700px" }}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
```

Comment:
We should remove `title` from the props passed down as it's not valid for a DataGrid so may result in warnings? You could destructure and spread the rest on line 13?